contracts/voting/dao-voting-token-staked/src/msg.rs

```diff
@@ -3,6 +3,7 @@ use cosmwasm_std::{Binary, Uint128};
 use cw_utils::Duration;
 use dao_dao_macros::{active_query, native_token_query, voting_module_query};
 use dao_interface::token::NewTokenInfo;
+#[allow(unused_imports)]
```

Comment:
Why were these needed?

Reply:
This cleans up the build output, because it doesn't recognize the Response is actually used in the QueryResponses:
![image](https://github.com/DA0-DA0/dao-contracts/assets/5104272/1e12bdb4-4189-4f9b-aa57-fdd7f8d94631)